scripts/install

```diff
@@ -65,9 +65,7 @@ ynh_script_progression --message="Installing dependencies..." --weight=7
 ynh_install_app_dependencies $pkg_dependencies
 tempdir="$(mktemp -d)"
 ynh_setup_source --dest_dir="$tempdir"
-ynh_package_update
-dpkg --force-confdef --force-confold -i /$tempdir/app.deb
-ynh_package_install -f
```

Comment:
Well why did you remove this line and the ynh_package_update ?
And why did you remove all dpkg parameters that I putt ?